CMakeLists.txt

```diff
@@ -25,21 +25,27 @@ if(${ENABLE_EPOLL_RECEIVE})
   add_definitions("-DENABLE_EPOLL_RECEIVE")
 endif(${ENABLE_EPOLL_RECEIVE})
 
-option(ENABLE_DOUBLE_RCVBUF "Enable double size of RCVBUF" OFF)
-if(${ENABLE_DOUBLE_RCVBUF})
-  add_definitions("-DENABLE_DOUBLE_RCVBUF")
-endif(${ENABLE_DOUBLE_RCVBUF})
+option(ENABLE_MODIFY_RECVBUF "Enable modify size of RECVBUF" ON)
```

Comment:
From the driver docs: `ENABLE_MODIFY_RECVBUF=OFF means No. This is the default.`

Oh lord... we should do this synchronization more often!

Reply:
What is this ?

Reply:
I'm writing a full summary of the review of changes; worry not. I will shed some light in this dark situation...